### DISCUSSION TITLE
ScriptReference/Events.UnityEvent_1

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#514] [Notes]
# ScriptReference/Events.UnityEvent_1

https://docs.unity3d.com/ScriptReference/Events.UnityEvent_1.html

**Comment by vertxxyz** [comment]
You haven't needed to override the class type since 2020.1:
> Scripting: The serializer can now serialize fields of generic types (e.g. MyClass someField) directly; it is no longer necessary to derive a concrete subclass from a generic type in order to serialize it.
